src/modules/flight_mode_manager/tasks/Utility/Sticks.hpp

```diff
@@ -53,11 +53,20 @@ class Sticks : public ModuleParams
 	Sticks(ModuleParams *parent);
 	~Sticks() = default;
 
-	bool checkAndSetStickInputs();
+	// Checks for updated manual control input & updates internal values
+	bool checkAndUpdateStickInputs();
+
 	bool isAvailable() { return _input_available; };
+
+	// Position : 0 : pitch, 1 : roll, 2 : throttle, 3 : yaw
 	const matrix::Vector<float, 4> &getPosition() { return _positions; };
 	const matrix::Vector<float, 4> &getPositionExpo() { return _positions_expo; };
 
+	// Helper functions to get stick values more intuitively
+	float getPitch() const { return _positions(0); }
+	float getRoll() const { return _positions(1); }
+	float getThrottle() const { return -_positions(2); } // Convert Z-axis(down) command to Up-axis frame
```

Comment:
Throttle is a bit of a special case. Right now it's globally defined as value between [0, 1] with 0 down, 1 up. I intend to for consistency with the other axes change this in https://github.com/PX4/PX4-Autopilot/pull/15949/ to [-1, 1] with -1 down, 1 up. For getThrottle I would clearly define what you get and personally would expect to receive [0, 1].

If I'm not mistaken this function now gives us [-1, 1]. The only reason I'm bringing this up is because the same library should be reusable to e.g. give throttle on a plane or multirotor in Manual mode and then you usually need [0,1].